docs/team/chongcheekaiclarence.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Chong Chee Kai Clarence's Project Portfolio Page
+---
+
+### Project: SoConnect
+
+SoConnect is a desktop app for NUS SoC students to manage their contacts, optimised for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
+Given below are my contributions to the project.
+
+* **Code contributed**: [RepoSense link](https://github.com/AY2223S1-CS2103T-W15-1/tp/pulls?q=is%3Apr+author%3AChongCheeKaiClarence)
```

Comment:
The RepoSense link here is incorrect. Please refer back to the 2103T instructions. You can also refer to the other PPPs for what the correct link looks like.